pkg/controller/build/helpers_test.go

```diff
@@ -160,6 +160,8 @@ func TestCanonicalizePullSecret(t *testing.T) {
 
 			if testCase.expectCanonical {
 				assert.Contains(t, out.Name, "canonical")
+				assert.True(t, isCanonicalizedSecret(out))
+				assert.True(t, hasCanonicalizedSecretLabels(out))
```

Comment:
Similar to earlier question, wouldn't isCanonicalizedSecret(out) check for labels too?

Reply:
It would. Although in this case, since its a test, the added verbosity is nice as it explicitly validates this particular case.